qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -1062,3 +1064,35 @@ def from_symplectic(cls, z, x, phase=0):
         """
         base_z, base_x, base_phase = cls._from_array(z, x, phase)
         return cls(BasePauli(base_z, base_x, base_phase))
+
+    def _qw_anticommutation_graph(self):
```

Comment:
FYI: We use a numpy-based implementation in opflow for anticommutation graph.
https://github.com/Qiskit/qiskit-terra/blob/b32a53174cbb14acaf885d5ad3d81459b8ce95f1/qiskit/opflow/converters/abelian_grouper.py#L133

Reply:
We discussed this out of band and I suggested going with this implementation because it is easier to write and easier to read. But, it might be a good idea to benchmark the present implementation vs. the opflow implementation.

Reply:
Yes, it's good to check the performance. When I implemented the numpy-based one, it was 10x faster than the naive one with H20 and NH3. https://github.com/Qiskit/qiskit-aqua/pull/923

Reply:
I revised the script for aqua#923 to compare this PR and opflow.
https://gist.github.com/t-imamichi/803cae0e8a6f106e679f593bb877b471

I think the naive one is intuitive, but seems very slow.

LiH JW
Opflow AbelianGrouper
Elasped time: 0.12327499399998487

This PR
Elasped time: 91.45781766899995

Reply:
Do you mean that the grouping alone was 10x faster, or was some integrated process involving H2O that much faster ?

Reply:
The graph generation `_qw_anticommutation_graph` is the bottleneck. Both opflow and this PR use `rx.graph_greedy_color` for coloring.

Reply:
It looks like there are significant performance differences between the two approaches, so I think it is a good idea to switch this current implementation to the Numpy-based implementation that is in opflow.

Reply:
The current version is as fast as opflow.
```
LiHJW.txt  631 Pauli strings
==> opflow.txt <==
Elasped time: 0.1262493520000021

==> paulilist.txt (this PR) <== 
Elasped time: 0.07839875100000882
```

```
NH3JW 4973 Pauli strings
==> opflow.txt <==
Elasped time: 5.371186884000053

==> paulilist.txt (This PR) <==
Elasped time: 5.728427904
```